examples/astro/src/react/hooks/usePageAsset.ts

```diff
@@ -0,0 +1,22 @@
+import type { DotCMSPageAsset } from "../../types";
+import { client } from "../../utils/client";
+import { useEffect, useState } from "react";
+
+export const usePageAsset = (currentPageAsset: DotCMSPageAsset | undefined) => {
+  const [pageAsset, setPageAsset] = useState<DotCMSPageAsset | undefined>();
+
+  useEffect(() => {
+    client.editor.on("changes", (page) => {
+      if (!page) {
```

Comment:
Why could this event send an undefined page?

Reply:
Is just a failsafe